build.sbt

```
@@ -54,7 +54,8 @@ lazy val fs2 = module(project) in file("modules/fs2")
 lazy val generic = genericModule(project) in file("modules/generic") dependsOn `generic-base`
 lazy val `generic-base` = genericModule(project) in file("modules/generic-base")
 lazy val hadoop = module(project) in file("modules/hadoop")
-lazy val http4s = module(project) in file("modules/http4s")
+lazy val ip4s = module(project) in file("modules/ip4s")
+lazy val http4s = module(project) in file("modules/http4s") dependsOn ip4s
```

nit: please keep the list of modules sorted alphabetically.

```suggestion
lazy val http4s = module(project) in file("modules/http4s") dependsOn ip4s
lazy val ip4s = module(project) in file("modules/ip4s")
```